src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Pre-built dasboards
+tags:
+  - Query your data
+  - Explore and query data
+  - Dashboards
+  - Pre-built dashboards
+---
+
+import dashboardsDashboardIndex from 'images/dashboards_screenshot-full_dashboard-index.png'
+
+import dashboardsPrebuiltViewAll from 'images/dashboards_screenshot-full_prebuilt-view-all.png'
+
+import dashboardsPrebuiltSearch from 'images/dashboards_screenshot-full_prebuilt-search.png'
+
+
+Through the New Relic UI, you can access a collection of pre-built dashboards based on high levels of user interest and activity. These pre-built dashboards enhance quickstart dashboard discovery, speed up the installation process, and get to instantly visualize your data.
+
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the dashboards page.
+
+<img
+  title="Dashboards index"
+  alt="Dashboards index"
+  src={dashboardsDashboardIndex}
+/>
+
+<figcaption>
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index.
+</figcaption>
+
+
+## How to install pre-built dashboards [#use]
+
+To install a pre-built dashboard:
+
+1. From the dashboards page, search for the pre-built dashboard you want to customize.
+2. Click the pre-built dashboard for selecting it.
+3. Select the account for adding the dashboard and data you select.
+4. Follow the instructions you have in the **Setup data source** section. You can skip this step.
+5. Click **View dashboard** to start visualizing your data. You can modify your pre-built dashboard, see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/). You can change the name, the settings, add new content, add new widgets, refine the charts, etc.
+
+
+## How to view more templates [#more-templates]
+
+You can browse our full dashboard template catalog if you don't find the dashboard template you need. Click the **View All** link next to the Popular pre-built dashboards title to see and search for all the available pre-built dashboards. Or you can use these buttons <Icon name="fe-edit-2"/> and <Icon name="fe-edit-2"/> to pass through the last edited pre-built dashboards.
```

Comment:
You can browse our full dashboard template catalog if you don't find the dashboard template you need. -> Browse the full dashboard catalog to find dashboard templates for all the available integrations.

Reply:
to pass through the last edited -> to see the most recent